sirius/coding/test_image_coding.py

```diff
@@ -31,7 +31,7 @@ def test_full(self):
 
         image = Image.open(data)
         image = image_encoding.crop_384(image)
-        image = image_encoding.threshold(image)
+        # image = image_encoding.threshold(image)
```

Comment:
Any particular reason for removing this test?

Reply:
Uhhh, it was probably causing something to break and I forgot about it 😅 I'll give it a look!

Reply:
Reverted this in bcd45c3